src/ai/backend/manager/config.py

```diff
@@ -328,6 +334,32 @@
             t.Key("sample-rate", default=_default_pyroscope_config["sample-rate"]): t.Null
             | t.ToInt[1:],
         }).allow_extra("*"),
+        t.Key("reporter", default=_default_reporter): t.Dict({
+            t.Key("smtp", default=[]): t.List(
+                t.Dict({
+                    t.Key("name"): t.String,
```

Comment:
Just a question, what is this `name` for?

Reply:
How about `reporter-name`?

Reply:
Why do we need to name it reporter-name if it's a child of reporter?